test-results/.last-run.json

```diff
@@ -0,0 +1,4 @@
+{
+  "status": "failed",
+  "failedTests": []
+}
```

Comment:
Hi test results also should bé added to git ignore

Reply:
For sure!